daemon/albatrossd_utils.ml

```diff
@@ -69,6 +69,10 @@ let systemd_socket_activation =
     let doc = "Pass this flag when systemd socket activation is being used" in
     Arg.(value & flag & info [ "systemd-socket-activation" ] ~doc)
 
+let syslog =
+  let doc = "Pass this flag when syslog should be used for logging" in
```

Comment:
would it be helpful to add at the end: "e.g. when using inetd" ?

Reply:
addressed in 704d5c23fa94322158ce9d39a6e866242074c80a